src/megaclient.cpp

```diff
@@ -13712,7 +13712,7 @@ bool MegaClient::startxfer(direction_t d, File* f, DBTableTransactionCommitter&
                 dstime timeleft = dstime(overquotauntil - Waiter::ds);
                 t->failed(API_EOVERQUOTA, committer, timeleft);
             }
-            else if (d == PUT && ststatus == STORAGE_RED)
+            else if (d == PUT && ststatus == STORAGE_RED && isPrivateNode(f->h))
```

Comment:
Same than above, but for a transfer that has not started yet

Reply:
In this case we there's no existing transfer so the new transfer shouldn't contains any foreign target.